arelle/RenderingResolution.py

```diff
@@ -219,9 +233,28 @@ def childContainsOpenNodes(childStructuralNode):
             if childContainsOpenNodes(node):
                 return True
         return False
+    
+def checkLabelWidth(view, strctMdlNode, subtreeRels, checkBoundFact=False):
```

Comment:
Seems like duplicated code? Suggest using a util or something instead of duplicating